prow/pjutil/pjutil.go

```diff
@@ -278,7 +279,18 @@ func JobURL(plank config.Plank, pj prowapi.ProwJob, log *logrus.Entry) string {
 		_, gcsPath, _ := gcsupload.PathsForJob(gcsConfig, &spec, "")
 
 		prefix, _ := url.Parse(plank.GetJobURLPrefix(pj.Spec.Refs))
-		prefix.Path = path.Join(prefix.Path, gcsConfig.Bucket, gcsPath)
+
+		// extract bucket name from gcsConfig
+		var bucketName string
+		if strings.Contains(gcsConfig.Bucket, "://") {
```

Comment:
This needs tests. Also it might be simpler to always split and do the if/else branching based on the length of the split result

Reply:
Fixed. I'll add tests in another commit

Reply:
Added test